cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/data/partitioner/NotEnoughReplicasException.java

```diff
@@ -20,7 +20,7 @@
 package org.apache.cassandra.spark.data.partitioner;
 
 import java.math.BigInteger;
-import javax.validation.constraints.NotNull;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Was this intended?

Reply:
Yes, we use `org.jetbrains.annotations.NotNull` elsewhere and `javax.validation.constraints.NotNull` is no longer depend on Jakata in the new common module.